charts/pravega/templates/version_map.yaml

```diff
@@ -0,0 +1,17 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "versionmap.fullname" . }}
+data:
+  0.1.0: "0.1.0"
+  0.2.0: "0.2.0"
+  0.3.0: "0.3.0,0.3.1,0.3.2"
+  0.3.1: "0.3.1,0.3.2"
+  0.3.2: "0.3.2"
+  0.4.0: "0.4.0"
+  0.5.0: "0.5.0,0.5.1,0.6.0,0.6.1,0.6.2,0.7.0"
+  0.5.1: "0.5.1,0.6.0,0.6.1,0.6.2,0.7.0"
+  0.6.0: "0.6.0,0.6.1,0.6.2,0,0.7.0"
```

Comment:
typo in line "0.6.0: "0.6.0,0.6.1,0.6.2,**0**,0.7.0""

Reply:
fixed